Features Section of the Wechaty landing page

encodeArnab opened the pull request:
Issue Number ( #912 )
This commit comprises of an atomic PR of Second section (Features Section with CSS) for the Wechaty landing page.

Images of hero section running locally

(Desktop Version)
<img width="618" alt="Screenshot 2021-09-08 at 1 16 16 PM" src="https://user-images.githubusercontent.com/77114532/132526500-4ad182cc-8a43-45f1-a43b-4e20963199fa.png">

(Mobile Version)
<img width="346" alt="Screenshot 2021-09-08 at 1 15 14 PM" src="https://user-images.githubusercontent.com/77114532/132526562-1c84398d-1b7b-49fd-81e5-cd0060954f08.png">
<img width="347" alt="Screenshot 2021-09-08 at 1 15 27 PM" src="https://user-images.githubusercontent.com/77114532/132526573-e6961fe6-7843-44bc-9e79-ffd112a6d527.png">

Rohitesh-Kumar-Jain reviewed (commented):
Check my comment on #1267 : )

encodeArnab commented:
Hello @Rohitesh-Kumar-Jain 
I've applied contrast to identify between distinct elements of the landing page, as per the recommended designs. The contrast is a little mild to match the overall UI, but each section is easily distinguishable. 
In addition, I updated the proposed changes and removed classnames and other style elements.

huan commented on docusaurus/src/pages/features-section/features-section.tsx:
```diff
@@ -0,0 +1,124 @@
+              <img
+                alt="Check Circle Vector"
+                id="vector"
+                src="https://static.overlay-tech.com/assets/a633a787-7bc5-4b36-abe7-fbc1a8a48f1f.svg"
```
We need to save those images to local

Rohitesh-Kumar-Jain commented on docusaurus/src/pages/features-section/features-section.css:
```diff
@@ -0,0 +1,624 @@
+  overflow-x: hidden;
+}
+
+#number100 {
```
if you may add some decription about this number100 that would be great

Rohitesh-Kumar-Jain commented on docusaurus/src/pages/features-section/features-section.css:
```diff
@@ -0,0 +1,624 @@
+  letter-spacing: 0.1em;
+}
+
+#number100Two {
```
same as above

Rohitesh-Kumar-Jain commented on docusaurus/src/pages/features-section/features-section.css:
```diff
@@ -0,0 +1,624 @@
+  overflow-x: hidden;
+}
+
+#number10000 {
```
same as above

encodeArnab replied:
Hello @Rohitesh-Kumar-Jain 
Okay, to make them more descriptive, I'll write a comment.

Rohitesh-Kumar-Jain commented on docusaurus/src/pages/features-section/features-section.tsx:
```diff
@@ -0,0 +1,86 @@
+              <img alt="Check Circle Vector" id="vector" src="tick-mark.webp" />
+            </div>
+            <div id="checkCircle">
+              <img alt="Check Circle Vector" id="vector" src="tick-mark.webp" />
```
Thanks for automatically implementing suggestion across the PRs : )
Great work 🎉

encodeArnab replied:
Thank You : )

Rohitesh-Kumar-Jain reviewed (approved):
Thanks for using more descriptive name, when the names aren't descriptive enough add comments

sajenjeshan1222 reviewed (approved):
LGTM
